tools/feature_support_matrix.rb

```diff
@@ -0,0 +1,68 @@
+require 'csv'
+
+def usage
+  <<-USAGE
+    Usage:
+      `bin/rails r tools/feature_support_matrix.rb`
```

Comment:
Since i brought it up, I'll fix up some of the existing scripts in tools that do the bin/rails runner way.